diff -uNr tpb-0.6.1-orig/configure.in tpb-0.6.1/configure.in
--- tpb-0.6.1-orig/configure.in	2004-01-19 10:51:20.000000000 +0100
+++ tpb-0.6.1/configure.in	2004-05-17 23:28:00.601553104 +0200
@@ -28,7 +28,7 @@
 AC_CHECK_HEADERS([fcntl.h getopt.h stdio.h stdlib.h strings.h sys/stat.h sys/types.h unistd.h wait.h])
 
 # Languages which your application supports
-ALL_LINGUAS="da de ru"
+ALL_LINGUAS="da de ru pl"
 AM_GNU_GETTEXT
                                                                                 
 AC_OUTPUT([
